src/py42/sdk/queries/alerts/filters/alert_filter.py

```diff
@@ -5,22 +5,68 @@
 
 
 def create_contains_filter_group(term, value):
+    """Creates a :class:`FilterGroup` for filtering results where the value with key ``term``
+    contains the given value. Useful for creating ``CONTAINS`` filters that not yet supported
+    in py42 or programmatically crafting filter groups.
+
+    Args:
+        term: (str): The term of the filter, such as ``actor``.
+        value (str): The value used to match on.
+
+    Returns:
+        :class`FilterGroup`
```

Comment:
in order for linking to properly work, you need to spell out the full path to the class. If you start the name with `~`, only the last part of the path will be displayed, e.g. `:class:`~py42.sdk.SDKClient` will appropriately link to that class, but will only display the text `SDKClient`.

Reply:
These ones do because they are in a different file.